pf.subdomain.conf

```diff
@@ -0,0 +1,44 @@
+
+## Version 2021/10/28
+# make sure that your dns has a cname set for pf
+
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+
+    server_name pf.*;
+
+    include /config/nginx/ssl.conf;
+
+    client_max_body_size 0;
+
+    location / {
+        # enable the next two lines for http auth
+        #auth_basic "Restricted";
+        #auth_basic_user_file /config/nginx/.htpasswd;
+
+        include /config/nginx/proxy.conf;
+        include /config/nginx/resolver.conf;
+        set $upstream_app pf;
+        set $upstream_port 80;
+        set $upstream_proto http;
+        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+
+        proxy_hide_header X-Frame-Options; # Possibly not needed after Portainer 1.20.0
+    }
+
+    location /api/websocket/ {
+        # enable the next two lines for http auth
+        #auth_basic "Restricted";
+        #auth_basic_user_file /config/nginx/.htpasswd;
+
+        include /config/nginx/proxy.conf;
+        include /config/nginx/resolver.conf;
+        set $upstream_app pf;
+        set $upstream_port 5555;
```

Comment:
whats your docker-compose configuration of pf service ? do you expose tcp/5555 ? do you use the pf-socket service?

Reply:
That's an error on my part.  That is there in case that service is desired use by anyone but is not needed and can be commented out or removed. I do not have it open on my firewall for example.